core/templates/core/home_page.html

```diff
@@ -80,6 +87,9 @@ <h3 class="double-height"><a style="color: {{ article.font_style.text_colour.hex
                             {% if article.editors_pick %}
                                 <div class="editors-pick"><div>Editors Pick</div></div>
                             {% endif %}
+                            {% if article == self.most_popular_article %}
+                                <div class="editors-pick"><div>Most Popular</div></div>
```

Comment:
We could rename this class to a more generic name if we are going to have the same styling for the different types.